common/test/acceptance/pages/lms/video/video.py

```diff
@@ -356,9 +356,9 @@ def click_player_button(self, button):
 
         self.q(css=button_selector).first.click()
 
-        button_states = {'play': 'playing', 'pause': 'pause'}
-        if button in button_states:
-            self.wait_for_state(button_states[button])
+        # button_states = {'play': 'playing', 'pause': 'pause'}
+        # if button in button_states:
+        #     self.wait_for_state(button_states[button])
```

Comment:
I saw the following error: `Promise not satisfied: State is playing ...`. 
[The logs were:](https://github.com/edx/edx-platform/blob/anton/fix-flaky-tests/common/test/acceptance/pages/lms/video/video.py#L668): 
```
state is buffered, current video position is 0:10
state is paused, current video position is 0:12
state is paused, current video position is 0:12
...
```